### PR TITLE
Automatic commit: Move 'workzone-workshop1-access' from QA to Production

### PR DESCRIPTION
**Pull request by LindsayBert**

### CODE REVIEW

File: tutorials/workzone-enrich-5-integrate-card/workzone-enrich-5-integrate-card.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ primary_tag: software-product>sap-work-zone
 author_name: Lindsay Bert
 ---
 
-# Integrate a UI5 Integration Card to Your SAP Build Work Zone, advanced edition
-<!-- description --> Add a custom UI5 integration card to your workpage.
+#  you
```
Comment (MichaelCzcz) [inline tutorials/workzone-enrich-5-integrate-card/workzone-enrich-5-integrate-card.md:10]: It seems like your title is missing

File: tutorials/workzone-build-5-url-app/workzone-build-5-url-app.md [hunk 2]
```diff
@@ -8,18 +8,18 @@ author_name: Lindsay Bert
 author_profile: https://github.com/LindsayBert
 ---
 
-# Add a URL App to Your SAP Work Zone Page
-<!-- description --> Create a URL app and add it to a page in SAP Work Zone.
+# Add a URL App to Your Workpage in SAP Build Work Zone, advanced edition
```
Comment (MichaelCzcz) [inline tutorials/workzone-build-5-url-app/workzone-build-5-url-app.md:11]: Please apply title case to "advanced edition"

File: tutorials/workzone-build-2-workspace/workzone-build-2-workspace.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ author_profile: https://github.com/LindsayBert
 ## Prerequisites
 - You have an environment set up and you can access your SAP Build Work Zone, advanced edition.
 &nbsp;
-**Note that it is not available in a trial account.**
+**Note that SAP Build Work Zone, advanced edition is not available in a trial account.**
```
Comment (MichaelCzcz) [inline tutorials/workzone-build-2-workspace/workzone-build-2-workspace.md:17]: This would look better formatted as a note with the label "Important"

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/workzone-build-5-url-app/workzone-build-5-url-app.md (hunk 2 above)
Comment (LindsayBert) [reply tutorials/workzone-build-5-url-app/workzone-build-5-url-app.md:11]: Hi Michael - this is the brand name from Brand Voice and we've been told not to capitalize advanced addition

**Pull request merged**